app/controllers/sessions_controller.rb

```diff
@@ -2,12 +2,18 @@ class SessionsController < ApplicationController
   def callback
     auth = request.env['omniauth.auth']
 
-    user = User.find_by(google_uid: auth[:uid])
-    unless user
-      user = User.create(google_uid: auth[:uid], name: auth[:info][:name], email: auth[:info][:email], image_url: auth[:info][:image])
+    if auth.info['email'].split("@")[1] == ENV['RESTRICT_DOMAIN']
```

Comment:
__[rubocop]__
* Prefer single-quoted strings when you don't need string interpolation or special symbols. [:ref](https://github.com/bbatsov/ruby-style-guide#consistent-string-literals)